### ISSUE TITLE
Bug: Dragend Event contains old annotation coordinates

### ISSUE THREAD

**Issue by adelsamy** [hansemannn/ti.googlemaps#33]
Hi,
i using dargend event to get lat and long final to pin but i can't  get it because dragend Event don't update marker lat and long please help me i need get final lat and long when dragend event work 

thank you for help me 
Regards,

**Comment by yozef** [comment]
@adelsamy When the `dragend` event fires, doesn't it return you an `annotation` object?

**Comment by adelsamy** [comment]
@yozef yes, return annotation object but with same first lat and long

**Comment by yozef** [comment]
It seems to be implemented properly. Then the Google Maps SDK dosen't seem to return the end location.

**Comment by adelsamy** [comment]
mmm, Ok what your Suggest to get end location ?

**Comment by hansemannn** [comment]
Yes, just checked it. Google Maps does not return anything else. I cannot find any example code, maybe it is an issue in the Google Maps iOS SDK?

**Comment by hansemannn** [comment]
Ok got it. We search for the annotation inside our cache and return it. Because it is updated during dragging, its location needs to be updated and be returned afterwards. Will setup a fix for the next release, but that can take a week or two.

**Comment by hansemannn** [comment]
Ok, so I finally had some time to investigate the issue and provided a possible solution in this PR: https://github.com/hansemannn/ti.googlemaps/pull/51 @yozef Since you know how to build, can you verify/test this?

**Comment by yozef** [comment]
Sure

**Comment by hansemannn** [comment]
Try [ti.googlemaps-iphone-2.5.1](https://www.dropbox.com/s/k3rqw8t5jtmkoeh/ti.googlemaps-iphone-2.5.1.zip?dl=1).

**Comment by hansemannn** [comment]
*bump* - Please provide feedback on the above version, thanks. I will close the issue on next Monday otherwise.

**Comment by hansemannn** [comment]
@yozef ?

**Comment by yozef** [comment]
I haven't tried it yet... was trying to figure out the adding annotation why it doesn't "always" appear.

I will give this issue of drag event a try this weekend

**Comment by yozef** [comment]
Hi, I got a 404 from Drop box for this link: https://www.dropbox.com/s/k3rqw8t5jtmkoeh/ti.googlemaps-iphone-2.5.1.zip?dl=1

**Comment by yozef** [comment]
I had an older copy of 2.5.1 when it was released and tested against it.
The annotations are not displaying in order to test it :)
![simulator screen shot oct 12 2016 9 55 33 am](https://cloud.githubusercontent.com/assets/287256/19312813/1b0e94d0-9062-11e6-9387-b9bbc58f1abe.png)

**Comment by hansemannn** [comment]
Just pushed a lot of stuff to https://github.com/hansemannn/ti.googlemaps/pull/51, please try the updated PR. I'm also drafting a public 2.6.0 Beta [here|https://github.com/hansemannn/ti.googlemaps/releases].

**Comment by yozef** [comment]
Hi @hansemannn just tried the new 2.6.0 module, with the exact code of Examples/app.js (added the GoogleMaps Key), when I run the App, it crashes as it runs with the following in the console:
```
[WARN] :   Ti.GoogleMaps: MapView is deprecated since View in favor of 2.2.0
// The 4 Warnings Below gets repeated for each Annotation
[WARN] :   Oct 14 09:05:16  TiGoogleMaps[73822] <Error>: CGContextDrawImage: invalid context 0x0. If you want to see the backtrace, please set CG_CONTEXT_SHOW_BACKTRACE environmental variable.
[WARN] :   Oct 14 09:05:16  TiGoogleMaps[73822] <Error>: CGBitmapContextGetData: invalid context 0x0. If you want to see the backtrace, please set CG_CONTEXT_SHOW_BACKTRACE environmental variable.
[WARN] :   Oct 14 09:05:16  TiGoogleMaps[73822] <Error>: CGContextDrawImage: invalid context 0x0. If you want to see the backtrace, please set CG_CONTEXT_SHOW_BACKTRACE environmental variable.
[WARN] :   Oct 14 09:05:16  TiGoogleMaps[73822] <Error>: CGBitmapContextGetData: invalid context 0x0. If you want to see the backtrace, please set CG_CONTEXT_SHOW_BACKTRACE environmental variable.
```

**Comment by hansemannn** [comment]
Fixed in the latest 2.6.0 release :-)
